CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Fixed
 
+ - #2165, Fix json/jsonb columns should have type "object" in OpenAPI spec
```

Comment:
Needs to be adjusted without "object". And add yourself, as in the other entries.